builder/oracle/oci/driver_oci.go

```diff
@@ -19,6 +20,22 @@ type driverOCI struct {
 	context       context.Context
 }
 
+var retryPolicy = &common.RetryPolicy{
+	MaximumNumberAttempts: 10,
+	ShouldRetryOperation: func(res common.OCIOperationResponse) bool {
+		if res.Error != nil {
+			return true
```

Comment:
In general we'd suggest only retrying retriable errors (e.g. 409 / 5xx etc).

Reply:
Let's do 429, 501, and 503 as per official docs:

https://docs.oracle.com/en-us/iaas/Content/API/References/apierrors.htm